FactorioBlueprintStringRenderer/src/com/demod/fbsr/fp/FPRotatedSprite.java

```diff
@@ -73,6 +73,9 @@ public void createSprites(Consumer<Sprite> consumer, FactorioData data, double o
 			fileIndex = index / fileLength;
 			tileIndex = index % fileLength;
 		}
+		if (fileIndex >= filenames.get().size()) {
+			System.out.println("Warning: Trying to access sprite " + index + " in " + filenames.get().size() + " files");
+		}
```

Comment:
This warning is probably no longer necessary and I could remove it. I originally added it and used a breakpoint on this line to find the cause of the exception.